PluginCore/PluginCore/Resources/zh_CN.resx

```diff
@@ -6120,4 +6120,16 @@ AND - All conditions are met.</value>
     <value>The default column to group results by.</value>
 	<comment>Added after 5.2.1</comment>
   </data>
+  <data name="LintingHelper.Description.LintOnSave" xml:space="preserve">
+    <value>Runs the available linters when a file is saved.</value>
+	<comment>Added after 5.2.1</comment>
+  </data>
+  <data name="LintingHelper.Description.LintOnOpen" xml:space="preserve">
+    <value>Runs the available linters when a file is opened.</value>
+	<comment>Added after 5.2.1</comment>
+  </data>
+  <data name="LintingHelper.Label.RunLinters" xml:space="preserve">
+    <value>Run Linters</value>
+	<comment>Added after 5.2.1</comment>
```

Comment:
Made me wonder too, but @Neverbirth told me to put this in there. Maybe it was just a mistake. Could you clarify @Neverbirth ?